.github/workflows/enforce-changelog-entry.yaml

```diff
@@ -0,0 +1,11 @@
+name: "Enforce changelog entry"
```

Comment:
**Suggestion:**
```suggestion
name: "Update unreleased section in CHANGELOG"
```
Just to be abundantly clear that this is about the unreleased section and not pertaining to an OpenFGA release.